colors/index.ts

```diff
@@ -1,6 +1,104 @@
-export * from './elevation';
-export * from './deepBlue';
-export * from './grayBlue';
-export * from './neon';
-export * from './orange';
-export * from './red';
+import * as elevation from './elevation';
+import * as deepBlue from './deepBlue';
+import * as grayBlue from './grayBlue';
+import * as neon from './neon';
+import * as orange from './orange';
+import * as red from './red';
+
+import {
+  blackA,
+  blue,
+  blueA,
+  blueDark,
+  blueDarkA,
+  gray,
+  grayA,
+  grayDark,
+  grayDarkA,
+  green,
+  greenA,
+  greenDark,
+  greenDarkA,
+  purple,
+  purpleA,
+  purpleDark,
+  purpleDarkA,
+  slate,
+  slateA,
+  slateDark,
+  slateDarkA,
+  whiteA,
+} from '@radix-ui/colors';
+
+const allColors = {
```

Comment:
`allColors` naming is misleading to me, I would have expected all colors to be part of this object, rather than only our custom colors.

Reply:
Yeah, I agree and think `customColors`, as you mentioned, would be a good name